feat: jsdom-v13

wtho opened the pull request:
This PR introduces jsdom v13 as default test environment for jest-preset-angular. This is achieved by
* Adding [jest-environment-jsdom-thirteen](https://github.com/theneva/jest-environment-jsdom-thirteen) as dependency
* Adding `testEnvironment: 'jest-environment-jsdom-thirteen' to `jest-preset.js`
* Removing Web Storage API mocks from `example/jestGlobalmocks.ts`
* Adding instructions in README Troubleshooting to configure usage of jsdom v11 (one line in jest configuration)

I added tests in a second commit to confirm it works, but I think they do not have to end up in the repository, as they basically test, what is already tested in jsdom itself.


Closes #216

thymikee commented on README.md:
```diff
@@ -420,3 +420,16 @@ global.$ = global.jQuery = $;
+
+### Failing with Node < v6
+
+By default we use JSDOM v13, which requires Node v6. If you want to use Node in a lower version than 6 to run your tests, make sure to configure jest to use an older JSDOM version, either in your `jest.config.js`, `jest.config.json`, `package.json`:
```
Since v12 jsdom requires Node 8+: https://github.com/jsdom/jsdom/blob/master/Changelog.md#1200

wtho replied:
You are right, I got confused!

ahnpnl commented:
LGTM 👍

wtho commented:
In latest commit:
* Added change to changelog
* rephrased README and fixed wrong version
* removed web storage api tests, features are tested by jsdom anyway

thymikee commented:
Thanks @wtho! Releasing new v7 beta in a minute

thymikee commented:
Available on `next` tag :)
